app/views/groups/_group_card.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="card">
+
+  <div class="card-header">
+    <h6 class="card-title">
+      <%= link_to group.group_name, group %>    
+        <span>
+          <%= link_to new_membership_path, class: "btn btn-link btn-sm text-muted" do %>
+          <i class="bi bi-person-plus-fill"></i>
+          <% end %>
+
+          <%= link_to edit_group_path(group), class: "btn btn-link btn-sm text-muted" do %>
+            <i class="bi bi-pencil-fill"></i>
+          <% end %>
+
+          <%= link_to group, method: :delete, class: "btn btn-link btn-sm text-muted" do %>
+            <i class="bi bi-trash-fill"></i>
+          <% end %>
+        </span>
+    </h6>
+  </div>
+
+  <div>
+    <details > <!--TODO: I need to change this for a modal or dropdown-->
+      <summary> Show contacts </summary>
+      <ul class="list-group list-group-flush">
+        <% group.contacts.order(:last_name).each do |contact|%>
+        <li class="list-group-item">
+          <%= link_to "#{contact.name} #{contact.last_name}", contact%> uses <%= contact.communication_channel%> 
+            <span class="badge float-right"> 
+              <%= link_to membership_path(Membership.where(group_id: group.id, contact_id: contact.id).ids.first), method: :delete, class: "btn btn-link btn-sm text-muted" do %>
```

Comment:
You can use `find_by` instead of `where` since you only expect one Membership.

```suggestion
              <%= link_to membership_path(Membership.find_by(group_id: group.id, contact_id: contact.id).id), method: :delete, class: "btn btn-link btn-sm text-muted" do %>
```